mode/sparql/sparql.js

```diff
@@ -71,20 +79,19 @@ CodeMirror.defineMode("sparql", function(config) {
       stream.eatWhile(/[a-z\d\-]/i);
       return "meta";
     }
-    else {
-      stream.eatWhile(/[_\w\d]/);
-      if (stream.eat(":")) {
+    else if (PREFIX_START.test(ch) && stream.match(PREFIX_REMAINDER, false)) {
+        stream.match(PREFIX_REMAINDER)
```

Comment:
Calling `match` with `false` as second argument and then calling it again with the same regexp is exactly equivalent to calling it without a second argument, so it looks like this line could go and the `false` could be removed.

Reply:
Will change as suggested.